app-backend/db/src/test/scala/com/azavea/rf/database/PropTestHelpers.scala

```diff
@@ -305,34 +322,43 @@ trait PropTestHelpers {
 
   def fixupTaskFeaturesCollection(
       tfc: Task.TaskFeatureCollectionCreate,
-      project: Project
+      project: Project,
+      statusO: Option[TaskStatus] = None
   ) =
     tfc.copy(
       features =
-        tfc.features map { fixupTaskFeatureCreate(_, project) }
+        tfc.features map { fixupTaskFeatureCreate(_, project, statusO) }
     )
 
   def fixupTaskFeatureCreate(
       tfc: Task.TaskFeatureCreate,
-      project: Project
+      project: Project,
+      statusO: Option[TaskStatus] = None
   ): Task.TaskFeatureCreate =
     tfc.copy(
-      properties = fixupTaskPropertiesCreate(tfc.properties, project)
+      properties = fixupTaskPropertiesCreate(tfc.properties, project, statusO)
     )
 
   def fixupTaskPropertiesCreate(
       tpc: Task.TaskPropertiesCreate,
-      project: Project
+      project: Project,
+      statusO: Option[TaskStatus] = None
   ): Task.TaskPropertiesCreate =
-    tpc.copy(projectId = project.id, projectLayerId = project.defaultLayerId)
+    tpc.copy(
+      projectId = project.id,
+      projectLayerId = project.defaultLayerId,
+      status = statusO.getOrElse(tpc.status)
+    )
 
   def fixupProjectExtrasUpdate(
       labelValidateTeamCreate: (Team.Create, Team.Create),
       labelValidateTeamUgrCreate: (UserGroupRole.Create, UserGroupRole.Create),
       dbOrg: Organization,
       dbUser: User,
       dbPlatform: Platform,
-      dbProject: Project
+      dbProject: Project,
+      labelsO: Option[List[(UUID, String, UUID)]] = None,
```

Comment:
Similar to above `labels0` -> `labelsOption` and `labelGroups0` -> `labelGroupsOption`